jck/runtime.api/playlist.xml

```diff
@@ -157,12 +162,13 @@
 	<test>
 		<testCaseName>jck-runtime-api-java_lang-String</testCaseName>
 		<variations>
-			<variation>NoOptions</variation>
+			<variation>-Xjit</variation>
```

Comment:
Please change this back to NoOptions

Reply:
It's only added for personal build tests, will be removed before I merge.